.github/workflows/automatizar_build.yml

```diff
@@ -0,0 +1,27 @@
+name: Refresh Docker image
+
+on:
+  push:
+    paths:
+      - 'Dockerfile'
+      - 'go.mod'
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Log in DockerHub
+      uses: docker/login-action@v2
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }} 
+
+    - name: Build and push to docker
+      uses: docker/build-push-action@v3
+      with:
+        push: true
+        tags: manujurado1/sportsbar-iv:latest

```

Comment:
La etiqueta latest no es necesaria (mira [mi subrayado aquí](https://itnext.io/understanding-docker-image-tags-and-publishing-images-to-docker-hub-b7a4f900f201), se asigna automáticamente si simplemente es la última.

Reply:
Tiene sentido que esa etiqueta se asigne automáticamente si, y usar ese comando tags para añadir otros tags identificativos.
Muchas gracias! 
Me pongo manos a la obra con el 6